commit 5130af40a26956a1f0d2be2224c685e119e18fe1 (refs/changes/88/5488/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2019-02-06T01:00:35+00:00 (8 months ago)
    
    MANTA-4092 Binder loses race with Zookeeper on startup leading to detached zk session watchers
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/package.json b/package.json
index 45bcdc5..a9b24e3 100644
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "once": "1.4.0",
         "vasync": "2.2.0",
         "verror": "1.10.0",
-        "zkstream":"0.10.1"
+        "zkstream": "0.11.3"
     },
     "devDependencies": {
         "nodeunit": "0.11.2"
